test/cli/start-cli.js

```diff
@@ -20,8 +20,8 @@ function isPreBreak(output) {
   return /Break on start/.test(output) && /1 \(function \(exports/.test(output);
 }
 
-function startCLI(args, flags = []) {
-  const child = spawn(process.execPath, [...flags, CLI, ...args]);
+function startCLI(args, flags = [], spawnOpts) {
```

Comment:
nit: Could you add a default value (even if it's just ` = {}` or ` = undefined`)? It reads a bit awkward when an optional parameter is in the middle of non-optional ones.

```suggestion
function startCLI(args, flags = [], spawnOpts = {}) {
```